avaframe/out3Plot/plotUtilsCfg.ini

```diff
@@ -46,13 +46,13 @@ unitpfd = m
 unitpfv = $ms^{-1}$
 unitP = kPa
 unitFV = $ms^{-1}$
-unitRelTh = m
+unitrelth = m
```

Comment:
I think this is the right time too finally correct all these variable names and make them consistent: unitppr to unitPPR etc....

Reply:
this is already an issue #370